.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: Test
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  test:
+    name: Wake scala compile
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Wit Init
+      uses: sifive/wit/actions/init@v0.13.2
+      with:
+        additional_packages: git@github.com:sifive/environment-blockci-sifive.git::0.7.0
+
+    - name: Wake Typecheck
+      uses: sifive/environment-blockci-sifive/actions/wake@0.7.0
+      with:
+        command: -x Unit
```

Comment:
Actually, you don't need any of this fancy stuff. The default rule for actions/wake is '-x Unit' already. You don't need the additional_package if you are not going to run anything.

So:
```
steps:
- uses: sifive/wit/actions/init@v0.13.2
- uses: sifive/environment-blockci-sifive/actions/wake@0.7.0
```
is all you actually need here.

Reply:
Makes sense, still might be worth having names, what about:
```
steps:
- name: Wit Init
  uses: sifive/wit/actions/init@v0.13.2
- name: Wake Typecheck
  uses: sifive/environment-blockci-sifive/actions/wake@0.7.0
```

Reply:
Entirely up to you. What you had is fine. What you propose now is fine.